whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticIndexServiceIntegrationTest.java

```diff
@@ -27,13 +24,14 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+//TODO[MA]: Setting up gitlab to use test container was not working as docker:dind was not starting up properly. It requires some config changes to runners
```

Comment:
I would move this comment also into `AbstractElasticSearchIntegrationTest.java` and `AbstractElasticSearchLocalTest.java`